docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -554,10 +556,15 @@ public void pullImage(String imageName) {
 
     public String execStart(String containerId, String... commands) {
         ExecCreateCmdResponse execCreateCmdResponse = this.dockerClient.execCreateCmd(containerId)
-                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty().withCmd(commands)
-                .exec();
-        InputStream consoleOutputStream = dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty(false).withCmd(commands)
                 .exec();
+        try {
+            dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                    .exec(new ExecStartResultCallback(System.out, System.err)).awaitCompletion();
+        } catch (InterruptedException e) {
+            return "";
+        }
+        InputStream consoleOutputStream = dockerClient.copyArchiveFromContainerCmd(containerId, "/execStartTest.log").exec();
```

Comment:
Using PipedOutputstream got IOException: Pipe not connected. I just use ByteArrayOutputStream and looks like this:

```java
public String execStart(String containerId, String... commands) {
        ExecCreateCmdResponse execCreateCmdResponse = this.dockerClient.execCreateCmd(containerId)
                .withAttachStdout(true).withAttachStdin(true).withAttachStderr(false).withTty(false).withCmd(commands)
                .exec();

        OutputStream outputStream = new ByteArrayOutputStream();
        try {
            dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
                    .exec(new ExecStartResultCallback(outputStream, System.err)).awaitCompletion();
        } catch (InterruptedException e) {
            return "";
        }

        return outputStream.toString();
    }
```

containerless's test was ok but ftest throws errors:

```
Caused by: org.yaml.snakeyaml.parser.ParserException: while parsing a flow sequence
 in 'reader', line 5, column 20:
                  env: [TOMCAT_PASS=mypass, "CATALINA_O ... 
                       ^
expected ',' or ']', but got FlowMappingStart
 in 'reader', line 5, column 136:
     ... PTS=-Djava.rmi.server.hostname=${docker.tomcat.host} -Dcom.sun.m ... 
```

I have not seen `docker.tomcat.host` set in any xml. Am I missing something?